Data-Structures/src/test/java/datastructures/tree/FizzBuzzTreeTest.java

```diff
@@ -36,4 +36,34 @@
         assertEquals("FizzBuzz",tree.root.left.value);
     }
 
+    @Test public void testTraversal() {
+        //       15
+        //    15     7
+        //          9  10
+//        NodeString root =
+        NodeString root = new NodeString("15", new NodeString("15"), new NodeString("7", new NodeString("9"), new NodeString("10")));
+        FizzBuzzTree tree = new FizzBuzzTree(root);
+        assertEquals("[15, 15, 7, 9, 10]",tree.breadthPrint().toString() );
```

Comment:
Remember, try to avoid testing with `toString()` unless you are testing `toString()`. This test could be refactored like this:  `assertArrayEquals(new Integer[]{15, 15, 7, 9, 10}, tree.breadthPrint().toArray());`